monai/transforms/post/array.py

```diff
@@ -69,22 +87,39 @@ def __init__(self, sigmoid: bool = False, softmax: bool = False, other: Optional
         self.softmax = softmax
         self.other = other
 
-    def __call__(  # type: ignore # see issue #495
-        self, img, sigmoid: Optional[bool] = None, softmax: Optional[bool] = None, other: Optional[Callable] = None
-    ):
+    def __call__(self, data: Iterable, *args, **kwargs):
+        """
```

Comment:
@benjamin-gorman These comments are good, and could/should be added to the solution identified in #521   I would recommend manually copying them into the correct places and make separate documentation only PR.